### PR TITLE
Refactor so LastUpdateCleaner and cleanLocationNames clean the same data

### PR DESCRIPTION
**Pull request by brja3022**
LastUpdateCleaner now returns covidDF and cleanLocationNames now uses that as its input data #30

### CODE REVIEW

**Review by vinceecws — APPROVED** [review]
Will need to further refactor so that
1) Both cleanLocationNames and LastUpdateCleaner lie inside /tools/Tools.Queries as utility methods which take in an RDD and returns a DF.
2) /clean/cleanLocationNames.scala will be removed.
3) /tools/Tools.Queries will be moved into /queries/queryConfirmedCases.scala and the module will be renamed /queries/Queries.scala
4) All inside LastUpdateCleaner will be moved into /tools/Tools.scala

Basically, we will only maintain 2 utility files, which are /tools/Tools.scala and queries/Queries.scala
